server/lib/src/value.rs

```diff
@@ -774,6 +776,25 @@ pub struct Session {
     pub scope: SessionScope,
 }
 
+impl fmt::Debug for Session {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        let issuer = match self.issued_by {
+            IdentityId::User(u) => uuid_to_proto_string(u),
```

Comment:
I think this needs to display "User - uuid-here" or "Sync - uuid-here". The current match is losing a lot of information about the issuer and the token that created the session.